atomic_reactor/plugins/pre_add_help.py

```diff
@@ -66,6 +68,29 @@ def run(self):
             result['status'] = self.NO_HELP_FILE_FOUND
             return result
 
+        dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
+        labels = dockerfile.labels
+
+        try:
+            help_file = open(help_path, 'r+')
+            lines = help_file.readlines()
+
+            if not lines[0].startswith("% "):
+                lines.insert(0, "%% %s (1) Container Image Pages\n" % labels.get("name") or "")
```

Comment:
Rather than `labels.get("name")` this should use `get_preferred_label(labels, 'name')`.